packages/react-components/react-breadcrumb/src/components/BreadcrumbButton/useBreadcrumbButtonStyles.ts

```diff
@@ -40,11 +40,33 @@ const useStyles = makeStyles({
   },
 });
 
+const useIconStyles = makeStyles({
+  small: {
+    fontSize: '12px',
```

Comment:
Is there any requirement to have the icon size to be overridden by the partners? If that's the case, it might be nice to introduce CSS variables for the font-size, height, and width, as they match. With this, it would be much simpler to override the styles on their side.

Reply:
I think it's better to have it. I moved it to CSS variables